import/api/tests/test_api.py

```diff
@@ -0,0 +1,101 @@
+"""Tests
+"""
+
+import logging
+
+from django.test import TestCase
+from model_mommy import mommy
+from rest_framework.reverse import reverse
+
+from datasets.blackspots import models
+from datasets.blackspots.models import Spot, Document
+
+log = logging.getLogger(__name__)
+
+
+class TestAPIEndpoints(TestCase):
+    """
+    Verifies that browsing the API works correctly.
+    """
+
+    def setUp(self):
+        for _ in range(3):
+            mommy.make(Spot)
+
+        self.spot_with_docs = mommy.make(Spot)
+        for _ in range(3):
+            mommy.make(Document, spot=self.spot_with_docs)
+
+    def valid_response(self, url, response):
+        """
+        Helper method to check common status/json
+        """
+
+        self.assertEqual(
+            200, response.status_code,
+            'Wrong response code for {}'.format(url))
+
+        self.assertEqual(
+            'application/json', response['Content-Type'],
+            'Wrong Content-Type for {}'.format(url))
+
+    def test_setup(self):
+        self.assertEqual(models.Spot.objects.count(), 4)
+        self.assertEqual(models.Document.objects.count(), 3)
+
+    def test_spot_list(self):
+        url = reverse('spot-list')
+
+        response = self.client.get(url)
+
+        self.valid_response(url, response)
+        data = response.data
+        self.assertEqual(len(data), 4)
+        spot_document_data = list(
+            filter(lambda spot: spot.get('locatie_id') == self.spot_with_docs.locatie_id, data)
```

Comment:
Can this be replaced with:
`[spot for spot in data if spot.get('locatie_id') == self.spot_with_docs.locatie_id]`

Reply:
ja, dat was inderdaad wat ik ook in gedachten had. Maar is dat echt leesbaarder?

Ik implementeer het wel